src/cljs/athens/effects.cljs

```diff
@@ -23,6 +27,10 @@
     (let [outputs (:tx-data (transact! db/dsdb datoms))]
       (pprint outputs))))
 
+      ;(def r (.. stream -Readable (from (dt/write-transit-str @db/dsdb))))
+      ;(def w (.createWriteStream fs "./data/my-db.transit"))
+      ;(.pipe r w))))
```

Comment:
Started off by doing streams instead of writing entire files synchronously. Premature optimization, but will probably want to do this eventually. Delete for now.